app/assets/stylesheets/application.scss

```diff
@@ -123,159 +123,235 @@
+blockquote:last-child {
+  margin-bottom: 0;
+}
+
+.reflink a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.reply .filesize {
+  margin-left: 20px
+}
+
+.userdelete {
+  float: right;
+  text-align: center;
+  white-space: nowrap
+}
+
+.replypage .replylink {
+  display: none;
+}
 
 /* wakaba rip */
 html, body {
-         background:#FFFFEE;
-         color:#800000;
+  background:#FFFFEE;
+  color:#800000;
 }
+
 a {
-        color:#0000EE;
+  color:#0000EE;
 }
+
 a:hover {
-        color:#DD0000;
+  color:#DD0000;
 }
+
 .adminbar {
-        text-align:right;
-        clear:both;
-        float:right;
+  text-align:right;
+  clear:both;
+  float:right;
 }
+
 .logo {
-        clear:both;
-        text-align:center;
-        font-size:2em;
-        color:#800000;
-        width:100%;
+  clear:both;
+  text-align:center;
+  font-size:2em;
+  color:#800000;
+  width:100%;
 }
+
 .theader {
-        background:#E04000;
-        text-align:center;
-        padding:2px;
-        color:#FFFFFF;
-        width:100%;
-}
-.postarea {
+  background:#E04000;
+  text-align:center;
+  padding:2px;
+  color:#FFFFFF;
+  width:100%;
 }
+
 .rules {
-        font-size:0.7em;
+  font-size:0.7em;
 }
+
 .postblock {
-        background:#EEAA88;
-        color:#800000;
-        font-weight:800;
+  background:#EEAA88;
+  color:#800000;
+  font-weight:800;
 }
+
 .footer {
-        text-align:center;
-        font-size:12px;
-        font-family:serif;
+  text-align:center;
+  font-size:12px;
+  font-family:serif;
 }
+
 .passvalid {
-	background:#EEAA88;
-	text-align:center;
-	width:100%;
-	color:#ffffff;
+  background:#EEAA88;
+  text-align:center;
+  width:100%;
+  color:#ffffff;
 }
+
 .dellist {
-	font-weight: bold;
-	text-align:center;
+  font-weight: bold;
+  text-align:center;
 }
+
 .delbuttons {
-	text-align:center;
-	padding-bottom:4px;
+  text-align:center;
+  padding-bottom:4px;
 
 }
+
 .managehead {
-	background:#AAAA66;
-	color:#400000;
-	padding:0px;
+  background:#AAAA66;
+  color:#400000;
+  padding:0px;
 }
+
 .postlists {
-	background:#FFFFFF;
-	width:100%;
-	padding:0px;
-	color:#800000;
+  background:#FFFFFF;
+  width:100%;
+  padding:0px;
+  color:#800000;
 }
+
 .row1 {
-	background:#EEEECC;
-	color:#800000;
+  background:#EEEECC;
+  color:#800000;
 }
+
 .row2 {
-        background:#DDDDAA;
-        color:#800000;
+  background:#DDDDAA;
+  color:#800000;
 }
+
 .unkfunc {
-	background:inert;
-	color:#789922;
+  background:inert;
+  color:#789922;
 }
+
 .filesize {
-	text-decoration:none;
+  text-decoration:none;
 }
+
 .filetitle {
-	background:inherit;
-	font-size:1.2em;
-	color:#CC1105;
-	font-weight:800;
+  background:inherit;
+  font-size:1.2em;
+  color:#CC1105;
+  font-weight:800;
 }
+
 .postername {
-	color:#117743;
-	font-weight:bold;
+  color:#117743;
+  font-weight:bold;
 }
+
 .postertrip {
-	color:#228854;
+  color:#228854;
 }
+
 .oldpost {
-        color:#CC1105;
-        font-weight:800;
+  color:#CC1105;
+  font-weight:800;
```

Comment:
Alright, yeah, but still you could admit that this is an incremental improvement on the existing formatting.